### PR TITLE
Refactor script to bump version

### PR DESCRIPTION
**Pull request by jeanpimentel**

### CODE REVIEW

File: app/scripts/bump_version.groovy [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    return content
+}
+
+
+def void commit(int[] versions) {
+
+    def version = "${versions[0]}.${versions[1]}.${versions[2]}"
+
+    def git_add = "git add versions.gradle"
+    def git_commit = "git commit -m 'Increase version'"
```
Comment (ivamluz) [inline app/scripts/bump_version.groovy:48]: Shouldn't we git pull before commiting?

File: app/scripts/bump_version.groovy [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    def git_tag = "git tag -a $version -m $version"
+    def git_push = "git push origin $version"
+
+    print git_add
+    println git_add.execute().text
+
+    print git_commit
```
Comment (ivamluz) [inline app/scripts/bump_version.groovy:55]: Why do we need this print statements? Won't the println already include these info?

File: app/scripts/bump_version.groovy (hunk 1 above)
Comment (jeanpimentel) [reply app/scripts/bump_version.groovy:48]: The pipeline is configured to clean the workspace when started, so it is presumed to be update

File: app/scripts/bump_version.groovy (hunk 2 above)
Comment (jeanpimentel) [reply app/scripts/bump_version.groovy:55]: I put it to see, in the log, the commands being made. The other print is to see the result of each command.

File: app/scripts/bump_version.groovy (hunk 1 above)
Comment (ivamluz) [reply app/scripts/bump_version.groovy:48]: Got it.

File: app/scripts/bump_version.groovy (hunk 2 above)
Comment (ivamluz) [reply app/scripts/bump_version.groovy:55]: Got it.

**Pull request merged**